react/src/components/dropdown/SprkDropdown.js

```diff
@@ -233,7 +261,7 @@ class SprkDropdown extends Component {
                               choiceFunction(value);
                             }
                           }}
-                          {...rest}
+                          {...choicesRest}
                         >
                           <p className="sprk-b-TypeBodyOne">{content.title}</p>
                           <p className="sprk-b-TypeBodyTwo">
```

Comment:
If an `infoLine` isn't present in the `item.content` then an empty `<p>` still gets rendered. Let's add some checks to only render what's necessary.